docs/quickstart.mdx

```diff
@@ -549,4 +549,4 @@ The Quickstart guide gives a tiny taste of what TensorZero is capable of.
 We strongly encourage you to check out the guides on [metrics & feedback](/gateway/guides/metrics-feedback/) and [prompt templates & schemas](/gateway/create-a-prompt-template).
 Though optional, they unlock many of the downstream features TensorZero offers in experimentation and optimization.
 
-From here, you can explore features like built-in support for [inference-time optimizations](/gateway/guides/inference-time-optimizations/), [retries & fallbacks](/gateway/guides/retries-fallbacks/), [experimentation (A/B testing) with prompts and models](/gateway/tutorial/#experimentation), and a lot more.
+From here, you can explore features like built-in support for [inference-time optimizations](/gateway/guides/inference-time-optimizations/), [retries & fallbacks](/gateway/guides/retries-fallbacks/), [experimentation (A/B testing) with prompts and models](/experimentation/run-automated-ab-tests), and a lot more.
```

Comment:
I still don't like A/B testing because I think just A/B testing by itself is a hard undersell